tests/bluetooth/tester/src/btp_gap.c

```diff
@@ -30,7 +30,11 @@ LOG_MODULE_REGISTER(LOG_MODULE_NAME, CONFIG_BTTESTER_LOG_LEVEL);
 #define CONTROLLER_NAME "btp_tester"
 
 #define BT_LE_AD_DISCOV_MASK (BT_LE_AD_LIMITED | BT_LE_AD_GENERAL)
+#if defined(CONFIG_BT_EXT_ADV)
+#define ADV_BUF_LEN (sizeof(struct btp_gap_device_found_ev) + CONFIG_BT_EXT_SCAN_BUF_SIZE)
```

Comment:
I think the report can be larger than `CONFIG_BT_EXT_SCAN_BUF_SIZE`. When the report is not fragmented, the Host uses the driver buffer directly, so the report can be up to `BT_BUF_EVT_DISCARDABLE_SIZE` minus some number of header bytes.

On this code path:

https://github.com/zephyrproject-rtos/zephyr/blob/f2b6490dee30b38dfb0ee31902177491c35ebacb/subsys/bluetooth/host/scan.c#L659

Reply:
Hi @alwa-nordic    Thanks for your comment, I have modified the code to 2 *  CONFIG_BT_EXT_SCAN_BUF_SIZE, it should    
enough, can you help review again